packages/x-svg-converter/src/__tests__/svg-to-vue.spec.ts

```diff
@@ -89,7 +89,7 @@ describe('test SVG to Vue script', () => {
    * Generate the SVG files used in the tests.
    */
   function generateTestSVGs(): void {
-    const absoluteSourcePath = path.resolve(process.cwd(), sourcePath);
+    const absoluteSourcePath = path.resolve(__dirname, sourcePath);
```

Comment:
I would say that this `absoluteSourcePath` is not needed, the `sourcePath` is already absolute, it is generated above with `path.resolve` and it contains already the `svg` folder for the tests.
So I would use `sourcePath` directly instead of creating this variable.